internal/resources/constants.go

```diff
@@ -32,6 +32,8 @@ const (
 	DSLBVirtualServerSslCert = "hpegl_vmaas_load_balancer_virtual_server_ssl_cert"
 	DSDhcpServer             = "hpegl_vmaas_dhcp_server"
 
+	DSMorpheusDataSource = "hpegl_vmaas_morpheus_details"
```

Comment:
We are checking if the `hpegl_vmaas_morpheus_details` is the right name to use

Reply:
This fine

Reply:
Thanks Mahesh.